### DISCUSSION TITLE
Unable to locate package php7.3

### DISCUSSION THREAD

**Discussion by gsnick** [oerdnj/deb.sury.org#2003] [General]
Hello,
Does someone know how to fix:
E: Unable to locate package php7.3
E: Couldn't find any package by glob 'php7.3'
E: Couldn't find any package by regex 'php7.3'?
The OS is Ubuntu 18.04.6 LTS

The repository is added using 
add-apt-repository ppa:ondrej/php
apt update
apt install php7.3 and it writes that Unable to locate package php7.3.
That is a clean OS. 

Thank you.

**Comment by stpnkcrk** [comment]
#1922

**Reply by gsnick** [reply]
Thanks.
